src/core/addonManager.js

```diff
@@ -100,6 +102,26 @@ export function getAddon(
   return Promise.reject(new Error('Cannot check add-on status'));
 }
 
+export function hasAbuseReportPanelEnabled({
+  navigator,
+}: { navigator: PrivilegedNavigatorType } = {}) {
+  if (hasAddonManager({ navigator })) {
+    const _navigator = navigator || window.navigator;
+    return _navigator.mozAddonManager.abuseReportPanelEnabled;
+  }
+  return undefined;
+}
+
+export function reportAbuse(
+  addonId: string,
+  { _mozAddonManager = window.navigator.mozAddonManager }: OptionalParams = {},
+) {
+  if (_mozAddonManager || module.exports.hasAddonManager()) {
```

Comment:
Shouldn't we reuse `hasAbuseReportPanelEnabled()`? Can we call `reportAbuse()` if `hasAbuseReportPanelEnabled()` returns `false`?

Reply:
Yes. 🤦‍♂